adr/cli.py

```diff
@@ -43,7 +44,11 @@ def query_handler(args, remainder):
         if query not in queries:
             log.error("query '{}' not found!".format(query))
             continue
-        print(format_query(query, args, fmt=args.fmt))
+        data, query_id = format_query(query, args, fmt=args.fmt)
+        print(data)
+        if query_id:
+            url = "http://activedata.allizom.org/tools/query.html#query_id={}".format(query_id)
```

Comment:
This should be based on the `ACTIVE_DATA_URL` (so if someone specifies --url, then the query url will also be updated). You can put it in a similar global in `query.py`, e.g:

    ACTIVE_DATA_QUERY_URL = "{}/tools/query.html".format(ACTIVE_DATA_URL)

Then you can append the `#query_url=<id>` part back in `cli.py` where you have it currently.